src/ubx.cpp

```diff
@@ -50,44 +50,15 @@
  * @see https://www.u-blox.com/sites/default/files/u-blox_ZED-F9P_InterfaceDescription_%28UBX-18010854%29.pdf
  */
 
-#include <assert.h>
-#include <math.h>
-#include <stdio.h>
-#include <string.h>
-#include <ctime>
-
 #include "ubx.h"
-#include "rtcm.h"
```

Comment:
Generally better to keep these in the source file. If they are in the header, the dependency set is unnecessarily larger if the header is included from several places.

The same holds for the more generic macros, like `MIN` or `SWAP16`, their scope should only be here.

Reply:
Thanks @bkueng, that's good advice. I've replaced the macros in the source file and deleted dependencies that were not needed for compilation.